indico/modules/events/abstracts/operations.py

```diff
@@ -39,15 +39,30 @@
 from indico.util.fs import secure_filename
 
 
+def _update_tracks(abstract, tracks, only_reviewed_for=False):
+    edit_track_mode = abstract.edit_track_mode
+    if edit_track_mode == EditTrackMode.none:
```

Comment:
I don't think we should do these types of check in the operation functions. I would move it to the RH.